app/models/dwelling.rb

```diff
@@ -12,19 +12,27 @@ class Dwelling < ApplicationRecord
 
   attr_accessor :audit_planning_application_id
 
-  TENURES = %w[open social intermediate].freeze
+  TENURES = %w[open social intermediate other].freeze
   TENURE_PRODUCTS = [
+    'Build for sale',
+    'Build to rent',
```

Comment:
Should this list be broken into sections based on those four Tenure categories? The first two appear to be "Open" and the last three "Social" based on the lower case headers in the zendesk ticket.

Reply:
Yeah, I was wondering about this too - this might need rethinking a bit...